docs/src/pages/[platform]/components/grid/react.mdx

```diff
@@ -1,7 +1,10 @@
-import { Grid, View } from '@aws-amplify/ui-react';
-
 import { GridDemo } from './demo';
-import { Example } from '@/components/Example';
+import {
+  DefaultGridExample,
+  GridResponsiveExample,
+  GridRowAndColumnSpanExample,
+} from './examples';
+import { Example, ExampleCode } from '@/components/Example';
 import { ComponentStyleDisplay } from '@/components/ComponentStyleDisplay';
 
 Grid primitive provides a grid container with style `display: grid`. Any of our other primitives can be used grid item children. To learn how to use Grid CSS properties, see the following documentation:
```

Comment:
A few minor tweaks: "Grid provides a CSS Grid container with style `display: grid`. Any Amplify UI components can be used as grid item children. To learn how to use CSS Grid properties..."